src/components/movie-card/movie-card.css

```diff
@@ -0,0 +1,48 @@
+.movie-card {
+  position: relative;
+  background-position: top;
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding-top: 160%;
+  border-radius: 10px;
+  color: white;
+}
+.title-container {
+  position: absolute;
+  height: 110px;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.7);
+  border-radius: 10px;
+}
+.swiper-pagination {
```

Comment:
Let's move all swiper styles to a global stylesheet instead. This way if we don't import the `movie-card` component somewhere but want the swiper, we won't have the adequate styles.

We have `index.css` let's move those there.

Let's remove these styles completely if we can fully control these styles with component props, otherwise do what I've said above.